datacenterlight/views.py

```diff
@@ -444,22 +444,37 @@ def post(self, request, *args, **kwargs):
         billing_address_data = request.session.get('billing_address_data')
         billing_address_id = request.session.get('billing_address')
         vm_template_id = template.get('id', 1)
-        final_price = specs.get('price')
 
         # Make stripe charge to a customer
         stripe_utils = StripeUtils()
-        charge_response = stripe_utils.make_charge(amount=final_price,
-                                                   customer=customer.stripe_id)
-
-        # Check if the payment was approved
-        if not charge_response.get('response_object') and not charge_response.get('paid'):
-            msg = charge_response.get('error')
-            messages.add_message(self.request, messages.ERROR, msg, extra_tags='make_charge_error')
-            return HttpResponseRedirect(reverse('datacenterlight:payment') + '#payment_error')
-
-        charge = charge_response.get('response_object')
+        cpu = specs.get('cpu')
+        memory = specs.get('memory')
+        disk_size = specs.get('disk_size')
+        amount_to_be_charged = (cpu * 5) + (memory * 2) + (disk_size * 0.6)
+        plan_name = "{cpu} Cores, {memory} GB RAM, {disk_size} GB SSD".format(cpu=cpu,
```

Comment:
Attention: as far as I can see there is a plan name and a "plan statement" that the user will see on their credit card bill. It is probably also to solve this in a later task, because this will involve having a "VM id" - I am not sure if we have this one here already and we might also want to ensure that two requests cannot have the same "VM id" - so maybe better to accept it for the moment, but to "Add VM id to stripe plan" in a different ticket later.